src/StaticFAQ/ArticleFeedback/ScreenInput.js

```diff
@@ -96,13 +109,17 @@ class ScreenInput extends React.Component<Props, State> {
           <div className="question">
             <Text>What problem were you trying to solve?</Text>
           </div>
-          <div className="inputArea">
+          <div className={`inputArea ${error ? 'invalid' : ''}`}>
             <textarea
               data-gramm_editor="false"
               onChange={this.handleChange}
               value={this.state.comment}
-              required
             />
+            {error ? (
+              <Typography size="small" type="error">
```

Comment:
`Typography` is deprecated, we should use `Text` or `Heading` now

Reply:
oh😓, thanks a lot for the feedback!